package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "eoapi-cdk",
-  "version": "9.1.1",
+  "version": "9.2.0-rc.3",
```

```suggestion
  "version": "9.1.1",
```

Needed to set this for publishing release candidates...